### ISSUE TITLE
breakmer removes arbitrary n regions

### ISSUE THREAD

**Issue by radaniba** [a-bioinformatician/BreaKmer#24]
I tested breakmer on small and large bed files (regions) and noticed that it removes or discards regions from the target file. 

I am not sure if it is a bug ?

Rad

**Comment by mducar** [comment]
Hi Rad,
Which version are you using?  The master branch should be considered unstable as it hasn't passed our internal QC and validation.  As such, we can't support that version.

If you are using the stable 0.0.6 version, can you provide a bit more detail?  Is there a pattern to the regions being removed?  How did you determine that regions were removed from analysis?

Thanks.

**Comment by radaniba** [comment]
not really, I don't know if there is a pattern. Actually in the output directory some regions are not created (subdierectories) so I am guessing breakmer is just skipping them

**Comment by ryanabo** [comment]
Hi Rad,

I believe those regions do not have any variants identified in the analysis. It removes temp files for space savings. Happy to help further if needed. Also, updated code is branched here: 

https://github.com/ryanabo/BreaKmer

Ryan
